crates/chain/src/descriptor_ext.rs

```diff
@@ -21,9 +21,9 @@ pub trait DescriptorExt {
     /// Panics if the descriptor wildcard is hardened.
     fn dust_value(&self) -> u64;
 
-    /// Returns the descriptor id, calculated as the sha256 of the descriptor, checksum not
+    /// Returns the keychain id, calculated as the sha256 of the descriptor, checksum not
     /// included.
```

Comment:
Docs need updating. It's now the sha256 of the spk at derivation index 0

Reply:
fixed.